controllers/alerts/alerts.go

```diff
@@ -27,6 +27,7 @@ const (
 	outOfBandUpdateAlert          = "KubevirtHyperconvergedClusterOperatorCRModification"
 	unsafeModificationAlert       = "KubevirtHyperconvergedClusterOperatorUSModification"
 	installationNotCompletedAlert = "KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert"
+	singleStackIPv6Alert          = "KubevirtHyperconvergedClusterOperatorSingleStackIPv6Alert"
```

Comment:
I see `KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert `, but the alerts should not be suffixed with the word `Alert`